src/main/java/org/apache/ibatis/jdbc/ScriptRunner.java

```diff
@@ -261,13 +260,10 @@ private void executeStatement(String command) throws SQLException {
           String message = "Error executing: " + command + ".  Cause: " + e;
           printlnError(message);
         }
-      }
-    } finally {
-      try {
-        statement.close();
-      } catch (Exception ignored) {
-        // Ignore to workaround a bug in some connection pools
-        // (Does anyone know the details of the bug?)
+      } catch (Exception e) {
+        // when close method throws some exception , it will catch by here .
+        // keep the original behavior unchanged
+        // see doc  https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
```

Comment:
Please remove this catch block (and the comment).
It's better to avoid catching unknown exception thrown from `execute()`.

Reply:
I have a question. How to solve the exception of close method.Hope reply soon

Reply:
As this class is an internal utility which is used only in tests, we can ignore exception from `close()`.